doc/source/examining/loading_data.rst

```diff
@@ -59,7 +59,7 @@ The functionality requires the package `ratarmount <https://github.com/mxmlnkn/r
 Under the hood, yt will mount the archive as a (read-only) filesystem. Note that this requires the
 entire archive to be read once to compute the location of each file in the archive; subsequent accesses
 will be much faster.
-All archive formats supported by `ratarmount <https://github.com/mxmlnkn/ratarmount>`_ should be loadable, provided
+All archive formats supported by `ratarmount <https://github.com/mxmlnkn/ratarmount>`__ should be loadable, provided
```

Comment:
Why does this need two underscores?

Reply:
the warning:
```
WARNING: Duplicate explicit target name: "ratarmount".
```
and the source of this solution (doubling the underscore): https://thomas-cokelaer.info/blog/2020/04/sphinx-raise-a-warning-duplicate-explicit-target-name/

Reply:
Ah, interesting.  OK, thanks.